gce/notebook/Dockerfile

```diff
@@ -2,17 +2,16 @@ FROM jupyter/base-notebook
 
 USER root
 RUN apt-get update \
-  && apt-get install -yq --no-install-recommends libfuse-dev nano fuse \
+  && apt-get install -yq --no-install-recommends libfuse-dev nano fuse vim git \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/*
 USER $NB_USER
 
 RUN conda install --yes -c conda-forge \
     cytoolz \
-    dask \
-    distributed \
+    dask=0.17.0 \
+    distributed=1.21.0 \
```

Comment:
just curious, what's the reason for pinning these versions

Reply:
1. These are newest
2. We should probably get in the habit of pinning more so that we're explicit about what was running when